src/components/context-menu/context-menu.module.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../styles/common/variables.module.scss';
+
+.content {
```

Comment:
It missing a border and a shadow.

Reply:
Haven't seen any in the figma, was it specified?

Reply:
<img width="1508" alt="Screenshot 2024-05-16 at 10 21 21" src="https://github.com/wixplosives/NotesApp/assets/31585545/30006f4e-2660-4481-befe-bef414b138e8">